build/cdc_services/Dockerfile

```diff
@@ -127,6 +127,7 @@ FROM gcr.io/datcom-ci/datacommons-services-runtime:2024-08-08 as runner
 # Env defaults.
 ENV WEBSITE_MIXER_API_ROOT=http://127.0.0.1:8081 \
     ENV_PREFIX=Compose \
+    FLASK_ENV=compose \
```

Comment:
Should this be "custom"?

Reply:
Duh - yes! Done - thanks for catching it.

Reply:
Also with the env variable set in the dockerfile, would users actually be able to override it?

Reply:
Yes. https://screenshot.googleplex.com/4XbKDwniTRgAsWm

Reply:
Looks like overriding should work (https://stackoverflow.com/questions/40334751/dockerfile-overriding-env-variable)

Reply:
awesome thanks!